File: Easy/0026-Remove-Duplicates-from-Sorted-Array/solution.cpp
// ═══════════════════════════════════════════════════════
//  Problem  : 0026. Remove Duplicates from Sorted Array
//  URL      : https://leetcode.com/problems/remove-duplicates-from-sorted-array/
//  Difficulty : Easy
//  Language : C++
//  Runtime  : 0 ms
//  Memory   : 8.2 MB
//  Solved   : August 27, 2026
// ═══════════════════════════════════════════════════════

class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int j = 0;
        for(int i = 1; i < nums.size(); i++) {
            if(nums[i] != nums[j]) {
                j++;
                nums[j] = nums[i];
            }
        }
        return j + 1;
    }
};

File: Easy/0026-Remove-Duplicates-from-Sorted-Array/solution.java
// ═══════════════════════════════════════════════════════
//  Problem  : 0026. Remove Duplicates from Sorted Array
//  URL      : https://leetcode.com/problems/remove-duplicates-from-sorted-array/submissions/2137820078/
//  Difficulty : Easy
//  Language : Java
//  Runtime  : 1 ms
//  Memory   : 46.8 MB
//  Solved   : September 10, 2026
// ═══════════════════════════════════════════════════════

class Solution {
    public int removeDuplicates(int[] nums) {
                int i = 0;
        for (int j = 1; j < nums.length; j++) {
            if (nums[j] != nums[i]) {
                i++;
                nums[i] = nums[j];
            }
        }
        return i + 1;
    }
}